lib/parsers/log-parser.js

```diff
@@ -76,10 +85,27 @@ export default class LogParser extends Parser {
 
       match = line.match(ERROR_PATTERN)
       if (match) {
+        let outMessageText
         const lineNumber = match[2] ? parseInt(match[2], 10) : undefined
+        // Check if the error is "Undefined control sequence"
+        let errMessageText = match[4]
+        if (errMessageText === UNDEF_CONTROL_SEQ_TEXT) {
+          // The actual macro is the last thing
+          // in the sentence of the next line
+          let nextLine = lines[index + 1]
+          let macroMatch = nextLine.match(LAST_MACRO_PATTERN)
+          // Check if there is a match
+          if (macroMatch) {
+            outMessageText = errMessageText + ': ' + macroMatch[1]
```

Comment:
Replace this with

``` js
outMessageText = `${errMessageText}: ${macroMatch[1]}`
```

Reply:
Whoop! Didn't know this was possible 🥇 🔢